anotherfile.js

```diff
@@ -0,0 +1,15 @@
+function calculate(operation, a, b) {
+  switch (operation) {
```

Comment:
a, b should be number
Suggestion: Pls add number check validati

**Suggestion:**
```suggestion
Pls add number check validation
```

**Context:**
```
function calculate(operation, a, b) {
```